resources/_gen/assets/scss/scss/tale.scss_5ad6f408b0e3e473c748aac88af0ea18.content

```diff
@@ -1,3 +1,33 @@
+:root.light {
```

Comment:
Where do these changes in `resources` come from?

Reply:
No idea. I will undo them.